LibreNMS/Alert/Transport/Hipchat.php

```diff
@@ -62,20 +62,7 @@ public function contactHipchat($obj, $option)
             $option["message_format"] = 'text';
         }
 
-        // Sane default of making the message color green if the message indicates
-        // that the alert recovered.   If it rebooted, make it yellow.
-        if (stripos($obj["msg"], "recovered")) {
-            $color = "green";
-        } elseif (stripos($obj["msg"], "rebooted")) {
-            $color = "yellow";
-        } else {
-            if (empty($option["color"]) || $option["color"] == 'u') {
-                $color = 'red';
-            } else {
-                $color = $option["color"];
-            }
-        }
-
+        $color = self::getColorForState($obj['state']);
```

Comment:
Does hipchat support hexcolors?

Reply:
As i see in documentation, it should.
Alternative an extension of `getColorForState` function to return named colors would be an option.
As PipoCanaja said, someone needs to test it. I also don't use Hipchat.
Slack i can test myself :-)

Reply:
ok, my fault. i looked on the false places.
Only a enum of colors is allowed.
I reverted this change.